app/src/main/res/layout/dialog_parking_configure.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/padding_parking_dialog_fragment"
+    android:paddingRight="@dimen/padding_parking_dialog_fragment"
+    tools:context=".MainActivity">
+
+    <TextView
+        android:id="@+id/title_fragment_dialog"
+        style="@style/TitleConfigureParkingDialogStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/text_configure_parking_dialog" />
+
+    <EditText
+        android:id="@+id/input_fragment_dialog"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/title_fragment_dialog"
+        android:inputType="number"
+        android:autofillHints=""
```

Comment:
this its necessary ?

Reply:
I putted this attribute because of a warning. It said that I must to add it even if my version is less than 26. I searched for a solution and said that I have to put autofillHints empty if I am not going to use.